lib/awful/spawn.lua

```diff
@@ -323,14 +323,12 @@ end
 --- Asynchronously spawn a program and capture its output.
 -- (wraps `spawn.with_line_callback`).
 -- @tparam string|table cmd The command.
--- @function callback Function with the following arguments:
+-- @tab callback Function with the following arguments
```

Comment:
> Should we rather use @tparam table for explicitness?

It's what I personally did everywhere so far for this reason. I also found it quite confusing when I started to work on the doc 18 months ago.